libyul/AsmAnalysis.h

```diff
@@ -74,23 +70,23 @@ class AsmAnalyzer: public boost::static_visitor<bool>
 		m_errorTypeForLoose(_errorTypeForLoose)
 	{}
 
-	bool analyze(assembly::Block const& _block);
-
-	bool operator()(assembly::Instruction const&);
-	bool operator()(assembly::Literal const& _literal);
-	bool operator()(assembly::Identifier const&);
-	bool operator()(assembly::FunctionalInstruction const& _functionalInstruction);
-	bool operator()(assembly::Label const& _label);
-	bool operator()(assembly::ExpressionStatement const&);
-	bool operator()(assembly::StackAssignment const&);
-	bool operator()(assembly::Assignment const& _assignment);
-	bool operator()(assembly::VariableDeclaration const& _variableDeclaration);
-	bool operator()(assembly::FunctionDefinition const& _functionDefinition);
-	bool operator()(assembly::FunctionCall const& _functionCall);
-	bool operator()(assembly::If const& _if);
-	bool operator()(assembly::Switch const& _switch);
-	bool operator()(assembly::ForLoop const& _forLoop);
-	bool operator()(assembly::Block const& _block);
+	bool analyze(Block const& _block);
+
+	bool operator()(yul::Instruction const&);
```

Comment:
Why do we need the extra scope operator only here?

Reply:
only in the cpp file. that one here was an old copy'n'paste from the beginning of the week. thx.